pkg/repository/v0manifest/version.go

```diff
@@ -54,8 +54,7 @@ func (v Version) IsEmpty() bool {
 
 // IsNightly returns true if the version is nightly
 func (v Version) IsNightly() bool {
-	return string(v) == version.NightlyVersion
-
+	return strings.Contains(string(v), "nightly")
```

Comment:
This is not true for v1 - although the nightly version should include `nightly` it doesn't have to.

Reply:
seems the only way we can check this is checking weather it contains "nightly" according to #261 @AstroProfundis

Reply:
I think if we put nightly in the component spec, then we don't need to check the version to know (from the manifests, whether a version is nightly is external to the version identifier).

Reply:
The idea in #261 was that, "nightly" is only a pointer to *some* version, and we have multiple such versions from daily builds but they are all treated as normal versions, except the one pointed by "nightly".

Reply:
A problem is that we must skip nightly versions when getting the latest version of a component, so we need to know which component versions are nightly and which aren't